### ISSUE TITLE
Support passing parameters to SQL inserts (to enable sanitation)

### ISSUE THREAD

**Issue by sebastianhaberey** [MarketSquare/Robotframework-Database-Library#146]
We would like to use the library to insert data into our database, like so:
```
Execute Sql String    INSERT INTO user (firstname, lastname) VALUES ('${first_name}', '${last_name}')
```
The problem is that this doesn't work for names like `Charles d’Artagnan`. It would be nice to have a parametrized SQL string instead:
```
Execute Sql String    INSERT INTO user (firstname, lastname) VALUES (%s, %s)    ${first_name}   ${last_name}
```
It looks like the method `execute_sql_string()` could be used as a base for a new method `execute_parameterized_sql_string()`.

**Comment by bhirsz** [comment]
Thank you for opening issue for it - it was implemented in linked PR and released in 1.4.1

**Comment by sebastianhaberey** [comment]
Awesome 👍
